src/NHibernate.Test/NHSpecificTest/GH1515/Fixture.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using NHibernate.Linq;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.GH1515
+{
+	[TestFixture]
+	public class Fixture : BugTestCase
```

Comment:
When not needing a session factory in the fixture, do not inherit from `BugTestCase` or `TestCase`.